_includes/api/en/5x/app-listen.md

```diff
@@ -43,7 +43,7 @@ The `app.listen()` method returns an [http.Server](https://nodejs.org/api/http.h
 
 ```js
 app.listen = function () {
-  var server = http.createServer(this)
+  const server = http.createServer(this)
```

Comment:
This is meant to be the exact code that is in the Express source tree: https://github.com/expressjs/express/blob/5f0c829d7ca7da746ee859f13a54631000f8a9b5/lib/application.js#L602-L605